doc/formats/default_parameters.rst

```diff
@@ -0,0 +1,84 @@
+.. _formatsdefault:
+
+Default parameters
+------------------
+
+Arbor supports reading default model parameters of cable-cells from JSON.
+These `default parameters` are applied at the global level of a model, affecting
+all simulated cable-cells. Some of these defaults can later be overridden at the
+cell level, which can also be done using JSON (see
+:ref:`decor formats <formatsdecor>` for more details.)
+
+The `default parameters` of a model are:
+
+   ========================================  =========================  =========
+   parameter                                 alias                      units
+   ========================================  =========================  =========
+   initial membrane potential                Vm                         mV
+   temperature                               celsius                    celsius
+   axial resistivity                         Ra                         Ω·cm
+   membrane capacitance                      cm                         μf⋅cm⁻²
+   initial internal concentration (per ion)  internal-concentration     mM
+   initial external concentration (per ion)  external-concentration     mM
+   initial reversal potential (per ion)      reversal-potential         mV
+   reversal potential method (per ion)       method                     --
+   CV policy                                 --                         --
+   ========================================  =========================  =========
+
+Of all possible default parameters, all but the CV policy can be specified using the
```

Comment:
Is there a reason for the exclusion of the discretisation policy (I know that NRN does not support it, but that's
no reason for us, right?)

Reply:
We can do it. Each `cv_policy` expects a different set of parameters, so we'd have to decide how to represent that, which shouldn't be too challenging, I just limited the scope of this PR of the agreed upon json format, we can always add `cv_policy` support later.

Reply:
Agreed.
@noraabiakar, can you provide a link to the ticket where you discuss extensions to what we have here (e.g. how to described discretisations).

Reply:
It's issue #1322. I'll add it to the PR description as well.